apps/meteor/app/apps/server/communication/methods.ts

```diff
@@ -74,13 +80,14 @@ export class AppMethods {
 			}),
 
 			'apps/go-disable': twoFactorRequired(function _appsGoDisable() {
-				if (!Meteor.userId()) {
+				const uid = Meteor.userId();
+				if (!uid) {
 					throw new Meteor.Error('error-invalid-user', 'Invalid user', {
 						method: 'apps/go-enable',
 					});
 				}
 
-				if (!hasPermission(Meteor.userId(), 'manage-apps')) {
+				if (!hasPermission(uid, 'manage-apps')) {
```

Comment:
For debug purposes, it will be better store the result of the hasPermission in a const